src/Type/Union/PostObjectUnion.php

```diff
@@ -41,7 +40,7 @@ public static function register_type( TypeRegistry $type_registry ) {
 	 *
 	 * @return array
 	 */
-	public static function get_possible_types() {
+	public static function get_possible_types(): array {
```

Comment:
I think it could be good to start adding return types throughout, but let's do that in a separate PR?